### ISSUE TITLE
inspectOnFailure does not trigger on failure

### ISSUE THREAD

**Issue by rollyboo** [vlucas/frisby#418]
I just switched to Frisby v2 from v1 and **inspectOnFailure** no longer works. When a test fails, Frisby doesn't dump request/response information. Do I miss something or this feature is not yet implemented in Frisby v2?
Thank you!

```
const frisby = require('frisby');

frisby.globalSetup({
	request: {
		inspectOnFailure: true,
		inspectRequestOnFailure: true // neither this
	}
});

it ('inspect on failure test', function (done) {
	frisby
		.get('http://google.com')
		.expect('status', 400) // make it fail
		.done(done);
});
```

**Comment by H1Gdev** [comment]
Hi @rollyboo 
In 2.x  inspectOnFailure is not supported.
(I do not know if inspectOnFailure will be supported in the future.)

**Comment by rollyboo** [comment]
Thank you @H1Gdev

**Comment by vlucas** [comment]
This was a feature in v0.x that was not ported over to v2. I would love to have it though if you have some time to look into re-adding it.

**Comment by vlucas** [comment]
This is now available (and the default) in `v2.0.14`.
